File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"
#include <string>
// TODO: add your solution here
namespace raindrops {
    std::string convert(int a) {
        std::string result = "";

        if (a % 3 == 0) result += "Pling";
        if (a % 5 == 0) result += "Plang";
        if (a % 7 == 0) result += "Plong";

        if (result.empty()) result = std::to_string(a);
        return result;
    }
}  // namespace raindrops

File: solutions/python/raindrops/1/raindrops.py
def convert(number):
    ans = ""
    if (number % 3 == 0):
        ans += "Pling"
    if (number % 5 == 0):
        ans += "Plang"
    if (number % 7 == 0):
        ans += "Plong"
    elif (number % 3 != 0 and number % 5 != 0 and number % 7 != 0):
        ans = str(number)
    return ans
